src/pages/home/report/MarkerBadge/MarkerBadgeContainer/MarkerBadgeContainerPropTypes.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Container Styles */
```

Comment:
NAB; I think it could be nice to make this "Styles to be assigned to Container" instead - slightly more verbose and more clear